pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.nuxeo</groupId>
+    <artifactId>nuxeo-parent</artifactId>
+    <version>2021.15</version>
+  </parent>
+
+  <groupId>org.nuxeo.compound.documents</groupId>
+  <artifactId>nuxeo-compound-documents-parent</artifactId>
+  <name>Nuxeo Compound Documents Parent</name>
+  <version>2021.0.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <description>Nuxeo Compound Documents allow binding several documents as parts of one complex document.</description>
+
+  <modules>
+    <module>nuxeo-compound-documents</module>
+    <module>nuxeo-compound-documents-package</module>
+  </modules>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>${project.groupId}</groupId>
```

Comment:
I would let `org.nuxeo.compound.documents`, more explcit.

Reply:
TLDR: :+1: 
More:
I see you and Thomas have the same pov. 
To me, saying project.groupId is faster to understand when reading diagonally "ok same project" while "org.nuxeo... wait is it exactly the same group? *scroll scroll or find occurences* oh yes exactly the same".

Reply:
Yes, only point of views, I guess we don't have any conventions, and rather stick to the common usage in the Nuxeo code base.